packages/terra-alert/themes/orion-fusion-theme/orion-fusion-theme.scss

```diff
@@ -1,41 +1,3 @@
 :global {
-  --terra-alert-color: #1c1f21;
-  --terra-alert-min-height: 3.41667rem;
-  --terra-alert-icon-font-size: 1.3333em;
-  --terra-alert-icon-margin-right: 0.475rem;
-  --terra-alert-icon-margin-left: 0;
-  --terra-alert-icon-margin-top: 0;
-  --terra-alert-body-margin-left: 0.5rem;
-  --terra-alert-body-margin-right: 0.5rem;
-  --terra-alert-body-margin-top: 0.75rem;
   --terra-alert-narrow-margin-right: 0.5rem;
```

Comment:
This is the only theme var that isn't used. Not sure if it's a miss or if we need to keep for backwards compatibility?

Reply:
I looked in the history and ... cannot find usage of it anywhere, past or present. 😕

Reply:
Shall we remove it?? I'm okay with it if everyone else is!

Reply:
Are we 100% sure? These aren't used in the consumer theme, clinical theme, lowlight or fusion themes?

Reply:
🤷‍♀️